router/network_router.go

```diff
@@ -230,29 +231,21 @@ func (router *NetworkRouter) ForgetConnections(peers []string) {
 }
 
 func (router *NetworkRouter) InitialPeers(peers []string) ([]string, error) {
-	var storedPeers []string
-	// Persistence disabled pending rationalisation of the expected behaviour
-	/*if _, err := router.db.Load(peersIdent, &storedPeers); err != nil {
-		return nil, err
-	}*/
-
-	if storedPeers != nil && !equal(peers, storedPeers) {
-		log.Println("Overriding initial peer list with stored list:", storedPeers)
-		peers = storedPeers
-	} else {
-		log.Println("Initial set of peers:", peers)
+	if _, err := os.Stat("/restart.sentinel"); err == nil {
```

Comment:
If we ever wanted to run weave non-containerised, this would be an awkward place to put that file. Perhaps the current dir would do?